packages/verkle/src/types.ts

```diff
@@ -123,3 +123,22 @@ export type FoundNodeFunction = (
 export const ROOT_DB_KEY = utf8ToBytes('__root__')
 
 export type VerkleCrypto = VerkleFFI
+
+export enum leafType {
```

Comment:
Short nit: We have our `enum`s capitalized by convention, so `leafType` -> `LeafType`.

Reply:
I've searched through the repo and the keys of the existing enums are also capitalized, will update both.